docs/man/docker-run.1.md

```diff
@@ -110,6 +111,14 @@ milliseconds.
 **--cidfile**=""
    Write the container ID to the file
 
+**--cpu-quota**=0
```

Comment:
Ah, ok, yes that makes sense. I saw your other doc regarding the multiple cpu cores.  Thank you for the quick explanation.